.travis.yml

```diff
@@ -29,7 +27,7 @@ jobs:
   fast_finish: true
 
 before_install:
-  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.19.1/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.19.1
+  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.1
```

Comment:
- We pin the version of the install script aiming for reproducible installs.
- We are stuck with v1.19 because of (still) open issues in golangci-lint.

The last commit on this file has the full details, see https://github.com/getsentry/sentry-go/commit/df74221585995c6f41deac769f9f9bb0e4d02c0c.

I'm afraid it is not time to bump the version just yet. See the `golangci-lint` build problem happened in this very PR: https://travis-ci.com/getsentry/sentry-go/jobs/279652187.

Reply:
I see it's related to `GO111MODULE=off` with `1.11` version. Let's keep this open for the right time.